src/components/app/DocumentList.js

```diff
@@ -186,7 +186,9 @@ class DocumentList extends Component {
             this.setState({
                 cachedSelection: null
             }, () => {
-                dispatch(setListIncludedView());
+                if (includedView.windowType === 'pickingSlot') {
```

Comment:
@wiadev  hmmm... not sure this one because 'pickingSlot' it's just one of the many windowTypes.
IMHO it shall work no matter which is the windowType.
If the process/action returned that "an included view shall be opened" then it shall be opened no matter what.

Btw, might be that i am not understanding the underlying problem from here. Pls feel free to discuss about that on chat or even voice call (preferable for me because it's faster).